app/code/Bulbulatory/Recommendations/Controller/Adminhtml/Recommendation/Delete.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Bulbulatory\Recommendations\Controller\Adminhtml\Recommendation;
+ 
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Bulbulatory\Recommendations\Model\Recommendation;
+ 
+class Delete extends Action
+{
+    protected $_model;
+ 
+    public function __construct(Context $context, Recommendation $model)
+    {
+        parent::__construct($context);
+        $this->_model = $model;
+    }
+ 
+    protected function _isAllowed()
+    {
+        return $this->_authorization->isAllowed('Bulbulatory_recommendations::recommendation');
+    }
+ 
+    public function execute()
+    {
+        $id = $this->getRequest()->getParam('recommendation_id');
+        $resultRedirect = $this->resultRedirectFactory->create();
+        if ($id) {
+            try {
+                $model = $this->_model;
+                $model->load($id);
```

Comment:
Nie wiem czy zwróciłeś uwagę ale `load` i `delete` są deprecated i oczywiście to rozwiązanie w tej wersji Magento będzie działać ale w kolejnych już niekoniecznie.

Reply:
Sugerowałbym wykonać akcje tak jak wskazano w tym artykule: http://vinaikopp.com/2017/02/18/magento2_repositories_interfaces_and_webapi/